redbot/cogs/cleanup/converters.py

```diff
@@ -15,18 +15,3 @@ async def convert(self, ctx: Context, argument: str) -> int:
             return int(argument)
 
         raise BadArgument(_("{} doesn't look like a valid message ID.").format(argument))
-
-
-PositiveInt = NewType("PositiveInt", int)
```

Comment:
You can remove the NewType import now as it is not used anywhere else in this file